### ISSUE TITLE
Support setting local variables

### ISSUE THREAD

**Issue by smpallen99** [nurugger07/calliope#39]
Support setting local variables with smart script per this example:
```
- var = "testing"
%p= var
```
It should generate the following EEx:
```
<% var = "testing" %>
<p><%= var %></p>
```
I'll have a pull request for this soon.

**Comment by nurugger07** [comment]
Awesome! :+1:

**Comment by nurugger07** [comment]
Closed with [PR #40](https://github.com/nurugger07/calliope/pull/40)
